argonaut/src/test/scala/org/http4s/argonaut/ArgonautSpec.scala

```diff
@@ -34,21 +34,21 @@ class ArgonautSpec extends JawnDecodeSupportSpec[Json] with Argonauts {
       val custom = ArgonautInstances.withPrettyParams(PrettyParams.spaces2)
       import custom._
       writeToString(json) must_== ("""{
-          |  "test" : "ArgonautSupport"
-          |}""".stripMargin)
+                                     |  "test" : "ArgonautSupport"
```

Comment:
Hmm, why is scalafmt happy with these changes?

Reply:
@aeons Not sure, I did a `;test;scalafmt` and it passed.

Reply:
I would prefer perhaps these be defined above and then checked against so the formatting could be cleaner. As that inset is not fun to read at the end of a test.